client/client.go

```diff
@@ -1924,17 +1922,16 @@ func (c *Client) removeAlloc(allocID string) {
 }
 
 // updateAlloc is invoked when we should update an allocation
-func (c *Client) updateAlloc(update *structs.Allocation) error {
+func (c *Client) updateAlloc(update *structs.Allocation) {
```

Comment:
What is the reason for no longer returning the error?

Reply:
We never did anything with update errors except log them, and now that updates are applied asynchronously there aren't even any errors to return.